### DISCUSSION TITLE
Momentum conservation equation of the liquid phase in the ScientificManual

### DISCUSSION THREAD

**Discussion by qiuqiubing** [Anura3D/Anura3D_OpenSource#103] [Q&A] [answered]
Dear all,
Regarding the liquid phase momentum conservation equation (Formula 3-26 on page 12) in the scientific manual, I have a question. Does it lack the term for the buoyant force's reaction?
![e63105d7ab4020f61d632935cdb0ed3](https://github.com/Anura3D/Anura3D_OpenSource/assets/48994907/c1a4a77a-3e83-45cd-9180-7e161e60e915)

**Comment by ayerrocolom** [comment, marked as answer]
This equation is correct.
